src/handlers/basic-handlers.ts

```diff
@@ -203,7 +203,7 @@ export function putRouteHandler(req: Request, res: Response, next: NextFunction)
 
   if (!id) {
     resLocals
-      .send(400, {
+      .send(INVALID_JSON, {
```

Comment:
Be careful here - you'd assigned meaning that wasn't there. This isn't being thrown because the JSON is invalid - it's being thrown because the `id` is required - it's a `BAD_REQUEST`.